sw/device/tests/sim_dv/uart_tx_rx_test.c

```diff
@@ -92,6 +122,110 @@ static volatile bool uart_irq_tx_empty_fired;
 static volatile bool exp_uart_irq_rx_overflow;
 static volatile bool uart_irq_rx_overflow_fired;
 
+// Convert arrays to integer value. Need to print array values to avoid being
+// optimized, so that TB can overwrite its values
+uint32_t bytes2word(uint8_t *bytes) {
```

Comment:
This should not be needed with my above suggestion.

Reply:
removed.